src/contexts/user-videos.jsx

```diff
@@ -2,29 +2,54 @@ import { createContext, useContext, useEffect, useReducer } from "react";
 import { getLikesArr } from "../utilities/likes-utils";
 import { getWatchLaterArr } from "../utilities/watchlater-utils";
 import { getAllVideos } from "../utilities/allvideos-utils";
+import { getPlaylists } from "../utilities/playlist-utils.jsx";
 const UserVideosContext = createContext();
 
 const userVideosReducer = (state, action) => {
```

Comment:
you can abstract this to another file so it look more cleaner and easy to debug (just for suggestion)